readme.txt

```diff
@@ -65,6 +65,10 @@ Web Stories are for everyone! If you are a site owner, content creator, or publi
 3. Explore our WYSIWYG editor with easy access to many editing tools (shown here: zooming and cropping media)
 4. Get started building your first story by editing one of our expressive, fully built template stories included with the editor.
 
+== Terms of Service ==
```

Comment:
This is actually not one of the default allowed headings. It needs to use `===` and moved right before `== Installation ==`, otherwise it won't be shown on the main page.